### DISCUSSION TITLE
"Rack Packs"  Backpack for large items only.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#6313] [Ideas and suggestions]
Add a new type of backpack which can hold multiple two-handed items. For instance, this new type of backpack could hold two coilgun ammo boxes, or three railgun shells, or three fire extinguishers or two suits.

**Comment by ViktorShield** [comment]
Barotrauma/Death Stranding crossover
Mudraptor Stranding

**Comment by ghost** [comment]
that would be kind of broken, no? what if you just held crates inside of it for a ton of storage?

**Comment by MaaiG** [comment]
I think it’s a good offer. That’s [one of the reasons](https://github.com/Regalis11/Barotrauma/discussions/10146)!

**Comment by MapleWheels** [comment]
Already exists in modded content, lookup Cargo Transport Mule. 90% of the time that you are carrying this many crates is during wreck salvages so the combo scooter + cargo container works.
